MyConfig.h

```diff
@@ -364,7 +364,7 @@
  */
 #ifdef MY_RX_MESSAGE_BUFFER_FEATURE
 #ifndef MY_RX_MESSAGE_BUFFER_SIZE
-#define MY_RX_MESSAGE_BUFFER_SIZE (20)
+#define MY_RX_MESSAGE_BUFFER_SIZE (10)
```

Comment:
Could we document why 10 is a better value than 20?

Reply:
MAX_MESSAGE_LENGTH is 32. So MAX_MESSAGE_LENGTH *20=640 ~1/3 of the meemory. I don't think so large queue is needed for nodes except the gate or repeater. May be reasonable to change this definition according the used platform.

Reply:
I agree that 10 should be sufficient for most nodes. Changing it based on platform would be nice, but maybe out of scope for this pull request (unless you feel like adding it).